web/src/app/user-profile/user-profile-form/user-profile-form.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="container">
+  <form (ngSubmit)="onSubmit()">
+
+    <div class="form-group">
+      <label for="firstName">
+        First name
+      </label>
+      <input id="firstName" name="firstName" class="form-control" type="text" [(ngModel)]="model.firstName">
+    </div>
+
+    <div class="form-group">
+      <label for="lastName">
+        Last name
+      </label>
+      <input id="lastName" name="lastName" class="form-control" type="text" [(ngModel)]="model.lastName">
+    </div>
+
+    <div class="form-group">
+      <label for="birthDate">
+        Birth date
+      </label>
+      <input id="birthDate" name="birthDate" class="form-control" type="date" [(ngModel)]="model.birthDate">
```

Comment:
**data-date-format="mm/dd/yyyy"**
or better (idk where u put it, but in initialization so that it is common for all calendars)
**$('.datepicker').datepicker({
    format: 'mm/dd/yyyy',
    startDate: '-3d'
});**

Reply:
Will be changed in the future if there is enough time

Reply:
Given solutions are not working or are not satisfying